tlsdefaults.go

```diff
@@ -11,9 +11,7 @@ var (
 	preferredCipherSuites = []uint16{
 		tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
 		tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
-		tls.TLS_ECDHE_RSA_WITH_RC4_128_SHA,
 		tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
-		tls.TLS_RSA_WITH_RC4_128_SHA,
```

Comment:
My only question here is what happens if we don't specify any cipher suites? The godocs say "TLS uses a list of suites supported by the implementation" in that case, but I really have little idea what that means.

I only ask because if the "suites supported by the implementation" also get an "A", then they would be useful because they're likely to have better cover traffic.